Realm/RLMObjectBase.mm

```diff
@@ -209,6 +209,18 @@ - (NSString *)descriptionWithMaxDepth:(NSUInteger)depth {
         if ([object respondsToSelector:@selector(descriptionWithMaxDepth:)]) {
             sub = [object descriptionWithMaxDepth:depth - 1];
         }
+        else if (property.type == RLMPropertyTypeData) {
+            NSData *data = object;
+            NSUInteger length = data.length;
+            if (data.length > 100) {
+                data = [NSData dataWithBytes:data.bytes length:100];
+                NSString *dataDescription = [data description];
+                sub = [NSString stringWithFormat:@"<%@ ... %lu more bytes>", [dataDescription substringWithRange:NSMakeRange(1, dataDescription.length - 2)], (unsigned long)(length - 100)];
```

Comment:
Instead of saying 'x more bytes' I think it would be more useful just to give the total size - 'x bytes total'